packages/shared/src/components/modals/streaks/StreakRecoverModal.tsx

```diff
@@ -0,0 +1,226 @@
+        fill="#FC538D"
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M9.5 67.5C23.5833 67.5 35 56.0833 35 42C35 27.9167 23.5833 16.5 9.5 16.5C-4.58326 16.5 -16 27.9167 -16 42C-16 56.0833 -4.58326 67.5 9.5 67.5ZM11.8164 27.8369L10.5354 26.1988C10.0182 25.5255 9.06804 25.4107 8.41308 25.9423C8.29856 26.0353 8.19774 26.1448 8.11376 26.2674C5.73163 29.7408 6.01715 33.9414 7.67405 37.0808L8.37921 38.4291L8.96525 39.576L9.43217 40.5215L9.67726 41.04L9.86942 41.469L10.0251 41.8916C10.4328 43.1655 10.309 44.4867 9.67804 45.6046C8.7245 47.2597 7.17421 47.651 5.71682 47.3998C3.33132 46.9885 2.60845 44.2696 2.38939 41.137L2.34292 40.3117L2.31473 39.4751L2.29672 38.2154L2.29415 36.9722L2.28664 36.6824C2.20653 35.2264 1.48552 35.2091 0.123611 36.6305C-3.69266 40.6151 -4.33545 46.7211 -1.47593 51.8778C0.672583 55.8099 5.02188 58.3933 9.47198 58.3937C9.66052 58.3937 9.85124 58.3889 10.0456 58.3791C14.6487 58.147 19.089 55.0977 21.0599 50.8149C22.1512 48.4775 22.5161 45.7788 22.0543 43.236C21.4539 39.9303 19.7257 37.7404 17.4219 34.8212C17.1519 34.479 16.874 34.1269 16.5891 33.7618L15.4705 32.3528L13.6891 30.1593L11.8164 27.8369Z"
+        fill="#FC538D"
+      />
+    </svg>
+  </div>
+);
+
+const StreakRecoverHeading = ({ days }: { days: number }) => (
+  <Typography
+    className="-mx-1 -mb-1 text-center font-bold"
+    tag={TypographyTag.H3}
+    type={TypographyType.Title1}
+    data-testid="streak-recover-modal-heading"
+  >
+    Oh no! Your
+    <span className="text-accent-bacon-default"> {days} day streak </span> has
+    been broken!
+  </Typography>
+);
+
+const StreakRecoveryCopy = ({
+  reputation,
+  cost,
+}: Record<'reputation' | 'cost', number>) => {
+  const isFree = cost === 0;
+  const canRecover = reputation >= cost;
+  const reputationLink = (
+    <a
+      target="_blank"
+      rel={anchorDefaultRel}
+      href={reputationHref}
+      title="What are reputation points?"
+      className="text-text-link underline-offset-2 hover:underline"
+    >
+      reputation points
+    </a>
+  );
+
+  return (
+    <Typography
+      className="text-center"
+      tag={TypographyTag.P}
+      type={TypographyType.Body}
+      data-testid="streak-recovery-copy"
+    >
+      {isFree && (
```

Comment:
maybe you can define these texts above and then conditionally render here

isFreeText, canRecoverText, noRecoverText